modules/fs.js

```diff
@@ -26,27 +22,26 @@
  *   console.error('test.txt already exists.');
  * }
  */
-
+var log = require("ringo/logging").getLogger(module.id);
 var arrays = require('ringo/utils/arrays');
+var {PosixPermissions} = require('ringo/utils/files');
 include('io');
 include('binary');
 
-var File = java.io.File,
-    FileInputStream = java.io.FileInputStream,
-    FileOutputStream = java.io.FileOutputStream;
+var Files = java.nio.file.Files,
+    FileSystems = java.nio.file.FileSystems;
+
+const FS = FileSystems.getDefault();
+const getPath = function(path) {
+    return FS.getPath(path);
+};
```

Comment:
Yep, this should be changed to `Paths.getPath(str)` or better `var getPath = java.nio.file.Paths.get;`